glean-core/rlb/src/core_metrics.rs

```diff
@@ -33,6 +33,7 @@ impl From<ClientInfoMetrics> for glean_core::ClientInfoMetrics {
             app_display_version: metrics.app_display_version,
             channel: metrics.channel,
             os_version: system::get_os_version(),
+            windows_build_number: Some(system::get_windows_build_number()),
```

Comment:
When the function returns `Option<T>` you will only need this:

```suggestion
            windows_build_number: system::get_windows_build_number(),
```